### PR TITLE
Task is not complated I will add fake api for using pagination correctly

### PR DESCRIPTION
**Pull request by fatima-28**

### CODE REVIEW

File: src/components/AllProductsCards/AllProductsCards.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+.product-single-card {
+    display: flex;
+    flex-direction: column;
+  cursor: pointer;
+  border: 1px solid #ebebeb;
```
Comment (Negodyaj) [inline src/components/AllProductsCards/AllProductsCards.scss:5]: you must use scss variables for colors

File: src/components/AllProductsCards/AllProductsCards.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './AllProductsCards.scss'
+import ProductImg from '../../images/productIphone.png'
+const AllProductsCards = () => {
```
Comment (Negodyaj) [inline src/components/AllProductsCards/AllProductsCards.tsx:4]: I'm not sure about the name of this component
from the html I see that it is a single card, but the name of component is plural...

File: src/components/AllProductsCards/AllProductsCards.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+      <div className="product-single-card">
+          <div className="single-card-item card-item-img">
+              <img src={ProductImg} className="card-image" alt="image" />
+          </div>
+          <div className="single-card-item card-item-context">
+              <div className="card-item-text-part">
+                  <p className="card-item-title card-text-part">Iphone 13 pro max</p>
+                  <p className="card-item-price card-text-part">19.99 $</p>
```
Comment (Negodyaj) [inline src/components/AllProductsCards/AllProductsCards.tsx:13]: you must take img src, title and price from props

File: src/components/images.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    const newOffset = (event.selected * itemsPerPage) % data.length;
+    
+    setItemOffset(newOffset);
+  };
+
+  return (
+    <>
+      <div className="main-products">
+          {currentItems.map(image=>{
+              return (
```
Comment (Negodyaj) [inline src/components/images.js:28]: you don't need "{" and "return (" after "=>"

File: src/components/images.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                <div className="product-single-card">
```
Comment (Negodyaj) [inline src/components/images.js:29]: every time you use .map to create several blocks of html you've got to set the "key" attribute
(https://reactjs.org/docs/lists-and-keys.html)

File: src/layouts/Products/All-Products/AllProducts.tsx [hunk 6]
```diff
@@ -1,9 +1,47 @@
...
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
+        <ProductCards />
```
Comment (Negodyaj) [inline src/layouts/Products/All-Products/AllProducts.tsx:40]: this must be done with help of some array

File: src/components/AllProductsCards/AllProductsCards.scss (hunk 1 above)
Comment (fatima-28) [reply src/components/AllProductsCards/AllProductsCards.scss]: I added variables for colors

File: src/components/AllProductsCards/AllProductsCards.tsx (hunk 2 above)
Comment (fatima-28) [reply src/components/AllProductsCards/AllProductsCards.tsx]: I change names as single item

File: src/components/images.js (hunk 4 above)
Comment (fatima-28) [reply src/components/images.js]: I removed it

File: src/components/images.js (hunk 5 above)
Comment (fatima-28) [reply src/components/images.js]: I used key attribute

File: src/layouts/Products/All-Products/AllProducts.tsx (hunk 6 above)
Comment (fatima-28) [reply src/layouts/Products/All-Products/AllProducts.tsx]: I change this part and get datas from our Api

File: src/components/AllProductsCards/AllProductsCards.tsx (hunk 3 above)
Comment (fatima-28) [reply src/components/AllProductsCards/AllProductsCards.tsx]: I get this card informations  from our Api from images.js ,thats why I deleted this AllProductsCards.tsx

File: src/components/images.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    const newOffset = (event.selected * itemsPerPage) % data.length;
+
+    setItemOffset(newOffset);
+  };
+
+  return (
+    <>
+      {currentItems.map((image) => (
+        <div className="product-single-card1">
+          <div className="single-card-item card-item-img" key={image.id}>
```
Comment (Negodyaj) [inline src/components/images.js:27]: you should add this key to div with "product-single-card1" class name =)

File: src/layouts/Products/All-Products/AllProducts.tsx [hunk 8]
```diff
@@ -1,11 +1,31 @@
...
-  return (
-    <section className='all-products'>
-      Hele ki kartlari yerlesdirmemisem, genisliyi bu qeder olacaq
-    </section>
-  )
-}
+    let products = document.querySelector(".main-products")
+    const [images, setImages] = useState([]);
+    useEffect(() => {
+        fetch("  http://localhost:3000/cards").then(
```
Comment (Negodyaj) [inline src/layouts/Products/All-Products/AllProducts.tsx:11]: it's better to have a constant with base url (in your case it's "http://localhost:3000")

File: src/components/images.js (hunk 7 above)
Comment (fatima-28) [reply src/components/images.js]: I fixed it

File: src/layouts/Products/All-Products/AllProducts.tsx (hunk 8 above)
Comment (fatima-28) [reply src/layouts/Products/All-Products/AllProducts.tsx]: I fixed it

**Pull request merged**